src/components/Authentification/EnterConfirmedPassword/EnterConfirmedPassword.css

```
@@ -0,0 +1,56 @@
+.password {
+    position: relative;
+}
+
+.enterConfirmedPassword__input{
+    background-color: #18825C;
+    opacity: 0.70;
+    color: white;
+    width: 300px;
+    border-radius: 10px;
+    margin-bottom: 10px;
+}
+
+.enterConfirmedPassword__input::placeholder {
+    color: white;
+}
+
+.enterConfirmedPassword__input:-webkit-autofill,
+.enterConfirmedPassword__input:-webkit-autofill:hover, 
+.enterConfirmedPassword__input:-webkit-autofill:focus {
+  border: none;
+  -webkit-text-fill-color: white;
+  transition: background-color 5000s ease-in-out 0s;
+  font-family: "Roboto", "sans-serif";
+}
+
+.enterConfirmedPassword__toggle__visibility {
```

Comme toggle et visibility ne sont pas des div différentes mais une seule, tu peux lier les deux mots avec un tiret :)
```suggestion
.enterConfirmedPassword__toggle-visibility {
```